valhalla/baldr/graphmemory.h

```diff
@@ -0,0 +1,24 @@
+#ifndef VALHALLA_BALDR_GRAPHMEMORY_H_
+#define VALHALLA_BALDR_GRAPHMEMORY_H_
+
+#include <memory>
```

Comment:
Whats this include for?

Reply:
You're right, it's not used and should be removed.